### PR TITLE
feat: prometheus exporter

### PR DESCRIPTION
**Pull request by muety**
Added a `/metrics` endpoint to be scraped by Prometheus.

### CODE REVIEW

**Comment by muety** [comment]
:warning: This PR is based on #9

**Comment by brentvollebregt** [comment]
My hosted version is currently running publicly on pythonanywhere; I feel this addition could potentially put a lot of unwanted stress on the application.

**Comment by muety** [comment]
I added a config flag to turn metrics on / off. They're disabled by default now.

**Comment by muety** [comment]
Please review again :slightly_smiling_face:

**Comment by brentvollebregt** [comment]
Looking good so far. We'll wait for #9 and then check back here on merge.

**Comment by muety** [comment]
I'll wait for #9 to be merged before attempting to resolve conflicts.

**Comment by muety** [comment]
Rebased onto latest master.

File: docs/config.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ The template of the SVG to respond with. Must take `count`, `width`, `recWidth`,
 ## `ENABLE_SSL`
 Enable SSL. Defaults to false for debugging purposes. Can be overridden with the `ENABLE_SSL` environment variable.
 
+## `EXPOSE_METRICS`
+Enable [Prometheus](https://prometheus.io) metrics at the `/metrics` endpoint. Defaults to false for performance reasons. Can be overridden with the `EXPOSE_METRICS` environment variable. However, use this feature with care. It is recommended only for moderately large deployments of _hit-counter_, as a separate metric will be exposed for every URL in your data base.
```
Comment (brentvollebregt) [inline docs/config.md:21]: > It is recommended only for moderately large deployments of _hit-counter_, as a separate metric will be exposed for every URL in your data base.  

Do you mean it's recommended for instances with low amounts of URLs stored?

**Review by brentvollebregt — CHANGES_REQUESTED** [review]

File: docs/config.md (hunk 1 above)
Comment (muety) [reply docs/config.md:21]: Exactly. Do you have doubts about it? Also, feel free to clarify that statement if you think it might be confusing.

Comment (brentvollebregt) [reply docs/config.md:21]: Sorry, I read

> It is recommended only for moderately large deployments of hit-counter

as saying it's only recommended to use this for instances with a large amount of data. I now see you are saying the instance needs to be large to hold all the URLs. My bad on this one.

**Comment by brentvollebregt** [comment]
@muety, thanks heaps again for this PR!

**Pull request merged**